### PR TITLE
Add some entries to .mailmap

### PR DESCRIPTION
**Pull request by JasonGross**
I ran `git shortlog -nse | sed s'/^[ 0-9\t]*//g' | sort | less` and
manually inspected the result for duplicates.

Then I used
```bash
git shortlog -nse | sed s'/^[ 0-9\t]*//g' | sort | sed s'/<.*>//g' | uniq -c | grep -v '^ *1 '
```
to check that there were no remaining duplicates.

**Kind:** infrastructure.

### CODE REVIEW

**Review by gares — COMMENTED** [review]
Not sure the canonical for Beta is the desired one

**Comment by JasonGross** [comment]
I chose the one listed on https://people.mpi-sws.org/~beta/, which his GitHub page lists as his homepage. 
But we can also ask him: @beta-ziliani, do you have a preference of canonical email address in .mailmap?

**Comment by beta-ziliani** [comment]
I have no idea what this is about, but in any case, beta@mpi-sws.org is my preferred email.

**Comment by ejgallego** [comment]
> I have no idea what this is about,

This is about what `git log` shows for your commits.

File: .mailmap [hunk 1]
```diff
@@ -87,6 +91,8 @@ Guillaume Melquiond <guillaume.melquiond@inria.fr> gmelquio <gmelquio@85f007b7-5
 Guillaume Melquiond <guillaume.melquiond@inria.fr> Guillaume Melquiond <guillaume.melquiond@gmail.com>
 Alexandre Miquel <miquel@gforge>                   miquel <miquel@85f007b7-540e-0410-9357-904b9bb8a0f7>
 Benjamin Monate <monate@gforge>                    monate <monate@85f007b7-540e-0410-9357-904b9bb8a0f7>
+Erik Martin-Dorel <erik.martin-dorel@irit.fr>      Erik Martin-Dorel <erik.martin-dorel@irit.fr>
```
Comment (Zimmi48) [inline .mailmap]: It seems to me that you are listing twice the same address on this line.

```suggestion
```

**Comment by Zimmi48** [comment]
>This is about what git log shows for your commits.

This is what `git log` shows when you pass it some options or when you have

```
[log]
        mailmap = true
```

in your `.gitconfig`.

**Review by ejgallego — APPROVED** [review]
@JasonGross can you fix the double-entry? I'll merge afterwards.

**Comment by ejgallego** [comment]
Well I'll take care of it, not worth the wait.

**Pull request merged**